cmake/Dependencies.cmake

```diff
@@ -96,8 +96,20 @@ if(BUILD_TEST)
     list(APPEND Caffe2_DEPENDENCY_LIBS ${Benchmark_LIBRARIES})
     include_directories(SYSTEM ${Benchmark_INCLUDE_DIRS})
   else()
+    # build as static library and disable install()
+    set(CAFFE2_OLD_BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS})
+    set(BUILD_SHARED_LIBS OFF)
+    macro (install)  # noop
+    endmacro ()
+
     add_subdirectory(${PROJECT_SOURCE_DIR}/third_party/benchmark)
     include_directories(SYSTEM ${PROJECT_SOURCE_DIR}/third_party/benchmark/include)
+
+    # restore static/dynamic setting and install()
+    set(BUILD_SHARED_LIBS ${CAFFE2_OLD_BUILD_SHARED_LIBS})
+    macro (install)
+      _install(${ARGV})
```

Comment:
Well, it was supposed to. But it looks like there are only 55 files in `install/` (instead of 761). So this doesn't work. My guess is `_install()` only existed in some particular version of CMake. Back to the drawing board ...